app/Http/Controllers/AuthController.php

```diff
@@ -397,6 +400,9 @@ public function passwordChangePost(Request $request)
             } elseif (strlen($pass_new1) < 10) {
                 $request->session()->flash('flash_message', 'Your new password should be at least 10 characters long.');
                 return view('auth.passchange');
+            } elseif ((new PwnedPasswords())->setPassword($pass_new1)->isPwnedPassword()) {
```

Comment:
I haven't been able to validate this with a quick Google: does this library automatically use the k-anonimity feature? As I don't see a configuration parameter for this specified anywhere.

Reply:
[Yes it does!](https://github.com/nickurt/laravel-pwned-passwords/blob/master/src/PwnedPasswords.php#L86)